toolchain/check/impl.h

```diff
@@ -23,7 +23,8 @@ auto ImplWitnessStartDefinition(Context& context, SemIR::Impl& impl) -> void;
 // Adds the function members to the witness for `impl`.
 auto FinishImplWitness(Context& context, SemIR::Impl& impl) -> void;
 
-// Sets all unset members of the witness for `impl` to the error instruction.
+// Sets all unset members of the witness for `impl` to the error instruction and
+// sets the witness id in the `Impl` to an error.
```

Comment:
Mirroring my comment above, if the current behavior is correct:

```suggestion
// Sets all unset members of the witness for `impl` to the error instruction and
// sets the witness id in the `Impl` to an error. Does nothing if there is no
// witness.
```

Reply:
I changed the behaviour instead